scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/WhitespaceTokenizer.scala

```diff
@@ -19,7 +19,14 @@ object WhitespaceTokenizer {
   def apply(input: Input, dialect: Dialect)(implicit
       options: Option[TokenizerOptions],
       tokens: java.util.Collection[Token]
-  ): WhitespaceTokenizer = new Grouping(input, dialect)
+  ): WhitespaceTokenizer =
+    if (options.forall(_.groupWhitespace)) new Grouping(input, dialect) else new Granular
```

Comment:
I don't mind to force custom options in Scalafix, but shouldn't `Granular` be the default to preserve the <4.9.8 behavior, at least in the 4.9.x line?

```suggestion
    if (exists(_.groupWhitespace)) new Grouping(input, dialect) else new Granular
```